Gemfile

```diff
@@ -1,10 +1,14 @@
 # frozen_string_literal: true
 
-source 'https://rubygems.org'
+source "https://rubygems.org"
 
 gemspec
 
 group :deployment do
-  gem 'rake'
-  gem 'rubocop'
+  gem "rake"
+  gem "rubocop"
```

Comment:
It is unclear why this is in the `deployment` group and not `test`. Is Rubocop run during deployment?